src/Movies/Movies.js

```diff
@@ -5,7 +5,7 @@ function Movies(props) {
   const moviesToDisplay = props.moviesList.map((movie, index) => {
     return (
       <section key={index} className="grid-item" id={movie.id.toString()}>
-        <img src={movie.poster_path}/>
+        <img alt='movie-poster-thumbnail' src={movie.poster_path}/>
```

Comment:
This is important for accessibility. I'll try to remember to throw these in the first time around!